main.tf

```diff
@@ -0,0 +1,95 @@
+resource "azurerm_resource_group" "launch" {
+  count    = var.create_resource_group ? 1 : 0
+  name     = var.resource_group_name
+  location = var.location
+}
```

Comment:
I'm trying to run `example/provision-aks-via-resource` but I am getting an error here:
```
│ Error: A resource with the ID "/subscriptions/636d899d-58b4-4d7b-9e56-7a984388b4c8/resourceGroups/beniscool-3252-rg" already exists - to be managed via Terraform this resource needs to be imported into the State. Please see the resource documentation for "azurerm_resource_group" for more information.
│ 
│   with module.launch.azurerm_resource_group.launch[0],
│   on ../../main.tf line 1, in resource "azurerm_resource_group" "launch":
│    1: resource "azurerm_resource_group" "launch" {
```

Reply:
I realized this is because I set `create_resource_group=false` in my `tfvars`. I didn't realize in the example that the variable was controlling whether the resource group would be created in another module.